src/throttler.guard.ts

```diff
@@ -139,6 +156,7 @@ export class ThrottlerGuard implements CanActivate {
     limit: number,
     ttl: number,
     throttler: ThrottlerOptions,
+    blockDuration: number,
     getTracker: ThrottlerGetTrackerFunction,
     generateKey: ThrottlerGenerateKeyFunction,
   ): Promise<boolean> {
```

Comment:
As we're already making a breaking change here, it would probably be beneficial to change all of these parameters into a single object parameter, so that adding new ones can be done optionally without changing the signature of the method. I may end up doing that after the PR, but wanted to get an initial thought on it